docs/dev/setup/integrated-code-lifecycle.rst

```diff
@@ -43,8 +43,10 @@ Create a file ``src/main/resources/config/application-local.yml`` with the follo
                use-external: false
            version-control:
                url: http://localhost:8080
-           # Only necessary on Windows:
            continuous-integration:
+               # Only necessary on ARM-based systems:
```

Comment:
I would add more details to the comment, in particular that Apple Silicon based systems such as M1, M2, M3 and ARM based server CPUs would need arm64 as architecture